config.yaml

```diff
@@ -4,7 +4,10 @@
 options:
   ranger-admin-password:
     description: |
-      Password used for the policy manager (Web interface)
+      The password for Ranger Admin user.
+      Password can not be changed using this property after initial deployment. 
```

Comment:
Do you have the logic implemented for checking if the password has been set initially to prevent re-setting it? If not, might be worth adding. I've implemented something similar for Temporal [here](https://github.com/canonical/temporal-k8s-operator/blob/45f4a36e5e0ec6a1190a14004103fa4e4da7f387/src/charm.py#L348).

Reply:
I do not. This is a good addition, thanks!